webservices/partition/base.py

```diff
@@ -105,7 +105,7 @@ def create_constraints(cls, child, cycle):
         start, stop = cycle - 1, cycle
         cmds = [
             'alter table {child} alter column {primary} set not null',
-            'alter table {child} alter column load_date set not null',
+            'alter table {child} alter column filing_form set not null',
```

Comment:
It's a non-null table constraint configured by the `fec_vsum_` tables, if you take that line out the tests fail as the child tabes aren't created with that matching constraint.